### PR TITLE
(Refresh) Administration: Change default site tagline to an empty string.

### PR DESCRIPTION
**Pull request by costdev**
This PR refreshes [PR 2230](https://github.com/WordPress/wordpress-develop/pull/2230) to fix failing tests and add another testcase as requested by @peterwilsoncc in [this comment](https://github.com/WordPress/wordpress-develop/pull/2230#discussion_r819245866).

Trac ticket: https://core.trac.wordpress.org/ticket/6479

### CODE REVIEW

File: tests/phpunit/tests/feed/atom.php [hunk 1]
```diff
@@ -63,6 +66,13 @@ public function set_up() {
 		$this->excerpt_only = get_option( 'rss_use_excerpt' );
 	}
 
+	/**
+	 * Tear down.
+	 */
+	public static function wpTearDownAfterClass() {
+		delete_option( 'blogdescription' );
```
Comment (costdev) [inline tests/phpunit/tests/feed/atom.php:73]: This fixes the failing tests.

File: tests/phpunit/tests/feed/rss2.php [hunk 2]
```diff
@@ -75,6 +79,13 @@ public function set_up() {
 		create_initial_taxonomies();
 	}
 
+	/**
+	 * Tear down.
+	 */
+	public static function wpTearDownAfterClass() {
+		delete_option( 'blogdescription' );
```
Comment (costdev) [inline tests/phpunit/tests/feed/rss2.php:86]: This fixes the failing tests.

**Comment by audrasjb** [comment]
Committed in Just another WordPress site

**Comment by audrasjb** [comment]
loool wrong copy paste 😂 😂 😂 

Committed in https://core.trac.wordpress.org/changeset/53815

**Pull request closed without merging**